addon_service/configured_addon/link/views.py

```diff
@@ -12,3 +21,19 @@ class ConfiguredLinkAddonViewSet(ConfiguredAddonViewSet):
         "authorized_resource",
     )
     serializer_class = ConfiguredLinkAddonSerializer
+
+    @action(
+        detail=True,
+        methods=[HTTPMethod.GET],
+        url_name="verified-link",
+        url_path="verified-link",
```

Comment:
Let's use `links` instead of `link` for URL `/configured-link-addons/<node guid>/verified-links`.